Docs/Generali/piano_di_progetto/02_modello.tex

```diff
@@ -1 +1,96 @@
+ai modelli agili, cercando di avvicinarsi il più possibile alle metodologie
+tipiche del framework \textbf{Scrum}. Questa scelta è motivata dalle seguenti
+considerazioni:
+\begin{itemize}
+    \item Scrum promuove la \textbf{collaborazione} continua tra i membri del team e gli
+          stakeholder;
+    \item Grazie agli sprint brevi, Scrum consente di \textbf{adattarsi} rapidamente a
+          nuove esigenze o priorità;
+    \item Gli sprint brevi consentono di individuare e \textbf{risolvere i problemi} più
+          velocemente;
+    \item Ad ogni sprint, il team si concentra su un \textbf{numero limitato di
+              attività}, aumentando l'efficienza;
+    \item I membri del team decidono come svolgere il lavoro durante gli sprint,
+          aumentando il senso di \textbf{responsabilità};
+    \item La natura iterativa di Scrum aiuta a suddividere grandi obiettivi in attività
+          più \textbf{piccole e realizzabili}.
+\end{itemize}
+
+\subsection{Ruoli Scrum}
+Scrum prevede tre ruoli (anche detti "responsabilità") che garantiscono che
+ogni aspetto del lavoro condiviso sia gestito in modo efficace:
+\begin{itemize}
+    \item \textbf{Sviluppatori}: lavorano insieme per creare qualsiasi
+          aspetto del prodotto. Le persone con qualsiasi competenza necessaria
+          per creare il prodotto assumono la responsabilità di sviluppatore. Nel
+          progetto didattico, tutti i membri del team ricoprono tale ruolo;
+    \item \textbf{Product owner}: sviluppa e comunica l'obiettivo del prodotto
+          ed è responsabile del product back-log. % trattino per andare a capo bene% 
+          Conosce e comprende il dominio e il
+          mercato del prodotto e vuole fornire agli utenti ciò di cui hanno bisogno.
+          Nel progetto didattico, il product owner è il proponente;
+    \item \textbf{Scrum master}: guida e dirige il team nell'adozione e
+          nella pratica di Scrum. In particolare cerca di massimizzare l'utilità
+          degli eventi e degli artefatti. Nel progetto didattico, lo Scrum master è
+          il membro del team con il ruolo di responsabile.
+\end{itemize}
+
+\subsection{Eventi Scrum}
+Gli eventi Scrum sono preziose opportunità per ispezionare e adattare il
+prodotto o il modo in cui il team lavora insieme.
+\begin{itemize}
+    \item \textbf{Sprint}: è un breve periodo di tempo in cui il team collabora per
+          completare una determinata quantità di lavoro. Lo sprint contiene tutti gli altri eventi.
+          Per il gruppo \textit{Six Bit Busters} ogni sprint dura due settimane;
+    \item \textbf{Sprint planning}: stabilisce l'obiettivo dello sprint.
+          Gli sviluppatori prevedono quale lavoro ritengono di poter realizzare durante
+          lo sprint per raggiungere l'obiettivo e come verrà completato il lavoro scelto.
+          In base all'obiettivo dello sprint e alla previsione, viene anche creato un piano iniziale;
```

Comment:
Scriverei:
stabilisce l'obiettivo dello sprint.
Gli sviluppatori prevedono quali lavori ritengono di poter realizzare durante
lo sprint per raggiungere l'obiettivo e come verrà completato il lavoro scelto.
In base all'obiettivo verrà successivamente creato un piano iniziale;